### PR TITLE
feat: convert jekyll migration to writr format

### PR DESCRIPTION
**Pull request by christianllv**

### CODE REVIEW

**Comment by christianllv** [comment]
@jaredwray actually is a simple copy post and images folders command. I did some refactor because the fileStorage doesn't need the Config parameters. 

I store the params as an object in config to run different code based on the parameters. If we want to implement a command we need to use command for the entire application. 

I will update this PR when test if everything is ok from your side. By the way, some files in Jekyll has .markdown extension; for this reason I will look .markdown files and rename to .md to be compatible with Writr.

**Comment by jaredwray** [comment]
@christianllv - I believe we can handle .markdown file extension and you should validate that.

**Comment by jaredwray** [comment]
@christianllv - can you get the build working and fix any bugs?

**Comment by christianllv** [comment]
> @christianllv - I believe we can handle .markdown file extension and you should validate that.

No, actually writer don't generate .markdown files. Please, let me know if you want to add compatibility to handle that files or we can change .markdown to .md file.

**Comment by christianllv** [comment]
> > > @christianllv - I believe we can handle .markdown file extension and you should validate that.
> > 
> > 
> > No, actually writer don't generate .markdown files. Please, let me know if you want to add compatibility to handle that files or we can change .markdown to .md file.
> 
> I believe it should as we are using ecto and it does support .markdown I thought.

No, I did some test and doesn't works.

**Comment by christianllv** [comment]
I will send test to cover 100% the code. Please, I need info about the .markdown to implement too. 

Thanks!

**Comment by jaredwray** [comment]
@christianllv - if something in writr is requiring .md can we make it so it handles both?

**Comment by christianllv** [comment]
> @christianllv - if something in writr is requiring .md can we make it so it handles both?

Yes, fixed now. I'm working on full cover test.

File: src/index.ts [hunk 1]
```diff
@@ -25,14 +26,17 @@ export class Writr {
...
 
         program.storeOptionsAsProperties(true);
-        
+
         program.option("-p, --path <path>", "Path of where the blog, config, and template are located");
         program.option("-o, --output <path>", "Path of where to output the generated blog");
         program.option("-r, --render <list>", "What do you want rendered such as html or json (example --render html,json)");
         program.option("-c, --config <path>", "custom configuration path");
-    
+        program.option("-j, --jekyll <path>", "Jekyll path to be migrated");
```
Comment (jaredwray) [inline src/index.ts:34]: @christianllv - are you thinking that we would have a command for each migration? Is that scalable as we are talking wordpress, medium, and ghostjs? Would it make more sense to have the command be `-m` or `--migrate` and then the next command be type and params?

```bash
writr --migrate jekyll <path> 
```

**Review by jaredwray — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (christianllv) [reply src/index.ts:34]: Totally agree.

My suggestion is writer --migrate jekyll <originPath> <destinationPath>

What do you think?

Comment (jaredwray) [reply src/index.ts:34]: I am good with that

Comment (christianllv) [reply src/index.ts]: Updated, please check

**Review by jaredwray — APPROVED** [review]

**Pull request merged**